### DISCUSSION TITLE
Can't connect to networks with WPA2-PSK

### DISCUSSION THREAD

**Discussion by ArchangelDesign** [PlummersSoftwareLLC/NightDriverStrip#34] [Q&A]
So far only tested on 2 devices but it appears that the router needs to either have open wifi or use WPA encryption. Anyone experiencing that?

**Comment by philcole** [comment]
My network is set to WPA2-PSK and it connects no problem. AP is a Netgear R7000

**Comment by davepl** [comment]
What other choice are you looking for?  On Oct 7, 2021, at 7:04 PM, Raff ***@***.***> wrote:So far only tested on 2 devices but it appears that the router needs to either have open wifi or use WPA encryption. Anyone experiencing that?—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by pathogen201** [comment]
I have a Heltec board that will not, but I think it may be the board. Adafruit Feather with the same build connects just fine. I am using an ASUS AX-11000 with WPA2 Personal PSK - AES.

**Comment by ArchangelDesign** [comment]
I'm using Tenda V15.03.06.50_multi (yeah, cheap stuff but has a lot of configuration capabilities)
I can't figure out why, but it appears to be the issue with this router. The device is found and a handshake is initiated but somewhere during that it crashes (I don't have a UART debugger so it's kinda a guess).
 In either case, all heavy lifting is done by the library so no need for any bug fixes here. I just need to invest in better hardware.

**Comment by altoidaddict** [comment]
I was doing fine until last night.  Initially I figured I was working on this so late that I somehow screwed my code up...after spending more than an hr trying to figure it out I went to bed.  Woke up today and tried to troubleshoot most of the day.  I have one M5 Plus stick from a few days ago that was working just fine and continues to work fine.  Therefore I decided just to wipe the problematic M5 C Plus stick, clear out my local code, and reimport a fresh copy of the repo and continue to have the problem....its driving me nuts.  Does this have anything to do with the amount of PSRAM?

after further review I also decided to upload the spectrum sketch to my heltec v2 and see the following in debug:
[E][WiFiGeneric.cpp:137] wifiLowLevelInit(): esp_wifi_init 4353
[E][WiFiSTA.cpp:120] begin(): STA enable failed!

ideas?
